.github/actions/has-auto-releasable-commits-v1/src/run.ts

```diff
@@ -0,0 +1,31 @@
+import shouldAutoRelease from './shouldAutoRelease'
+import type { Core, CommitList } from './types'
+
+export default async function run(core: Core) {
+  try {
+    const commits = core.getInput('commits', { required: true })
+    const versionLocked = core
+      .getInput('version-locked', {
+        required: true
+      })
+      .toLowerCase()
+
+    if (!['true', 'false'].includes(versionLocked)) {
+      core.setFailed(
+        `Invalid value for version-locked: ${versionLocked}. Must be true or false`
+      )
+      return
+    }
+
+    const commitList: Array<CommitList> = JSON.parse(commits)
+    const isVersionLocked = versionLocked === 'true'
+    const shouldRelease = shouldAutoRelease({
+      commitList,
+      isVersionLocked
+    })
+
+    core.setOutput('should-release', shouldRelease)
```

Comment:
We should `core.info` the result of `shoudlRelease` to make debugging easier

```suggestion
    core.info('Set should-release output: ' + shouldRelase)
    core.setOutput('should-release', shouldRelease)
```